pyproject.toml

```diff
@@ -14,7 +14,7 @@ readme      = ["README.rst", "docs/source/changelog.rst"]
 req2flatpak = 'req2flatpak:main'
 
 [tool.poetry.dependencies]
-python    = "^3.7.2"
+python    = "^3.8"
```

Comment:
Not sure we want to drop `3.7`. This is a very simple build script that should easily support most python versions. `py3.7` is still used in many distributions or on many installations afaik.

Updating `sphinx-rtd-theme` to `^1.2.0` already fixes the issue. No need to update `sphinx`.

Reply:
I agree. I'll re-do the dependency updating, keeping compatibility with py3.7

Reply:
done.